lib/cuke_cataloger/unique_test_case_tagger.rb

```diff
@@ -398,19 +398,14 @@ def update_rows_if_needed(test, sub_id, id_column_name)
     end
 
 
-    # Slowest way to get the id tag. Will check the object every time.
-    def current_id_tag_for(thing)
-      id_tag_for(thing)
-    end
-
     # Faster way to get the id tag. Will skip checking the object if an id for it is already known.
     def fast_id_tag_for(thing)
       @known_id_tags ||= {}
 
       id = @known_id_tags[thing.object_id]
 
       unless id
-        id = current_id_tag_for(thing)
+        id = id_tag_for(thing)
```

Comment:
As previously mentioned, this is the kind of place where I would rather see the more informative name of #current_id_tag_for`.